examples/TestAppUwp/SignalingPage.xaml.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace TestAppUwp
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed partial class SignalingPage : Page
```

Comment:
nit: fill comment, or make internal so we don't need boilerplate docs.

Reply:
Fixed.

Reply:
Note: DocFX doesn't run on TestAppUWP at the moment, so comments are technically not required, though they do provide IMHO some huge benefit for us in remembering how things work and for other users in learning those things. So I like to even comment on protected/private/internal items.